### DISCUSSION TITLE
Light isolation without signature spoofing

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3592] [Various ideas and suggestions]
Some programs that Sandboxie can run, such as Steam, block access to anti-cheating servers because they are unable to verify signatures. Is there a way to run Sandboxie for such situations, where some light isolation applies, but signature is not spoofed? Sandboxie obviously can't be expected to work with BattlEye-protected games because BattlEye requires a full access to memory via system driver it spawns during gameplay sessions, but Valve's anti-cheating is not driver-based.

**Comment by OliverTrust** [comment]
VAC work very strange in sandboxie. In CS2 i can start official matchmaking with no problems and will kicked in middle game by VAC session error. But when I tested Avast sandbox, VAC does not throw any errors at all and allow me to play full game
